.github/workflows/main.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  industrial_ci:
+    name: ${{ matrix.env.CI_NAME }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        env:
+          - {CI_NAME: Bionic-Build,
+              OS_NAME: ubuntu,
+              OS_CODE_NAME: bionic,
+              ROS_DISTRO: melodic,
+              ROS_REPO: main,
+              UPSTREAM_WORKSPACE: "github:ros-industrial-consortium/tesseract#master github:ros-industrial-consortium/trajopt_ros#master
```

Comment:
We should be able to do something like this to avoid building two versions of Trajopt.
```
             UPSTREAM_WORKSPACE: 'github:swri-robotics/descartes_light#master github:Jmeyer1292/opw_kinematics#master github:ethz-adrl/ifopt#master',
             TARGET_WORKSPACE: '. github:ros-industrial-consortium/tesseract#master',
```

Reply:
Yep thanks, I understood it a bit after, it's done in https://github.com/ros-industrial-consortium/trajopt_ros/pull/194/commits/85ee28ea02e17f981e721b8651c9ff8225b2a2a4

Reply:
Thanks. I didn't see your commits until I submitted this.

Reply:
It seems it works for both. Build result updated at https://github.com/brisa-robotics/trajopt_ros/runs/846092749?check_suite_focus=true